### DISCUSSION TITLE
Transparent Body Background

### DISCUSSION THREAD

**Discussion by zeknoss** [epicmaxco/vuestic-ui#4148] [Q&A] [answered]
Hi everone,
I've built a simple To Do app for my Homepage instance. I'm trying to make the background of the body transparent but no matter what I do, it does not work. I've tried other frameworks just to see if I'm not missing anything outside Vuestic and they all work without any issues.

Any ideas what could be the reason behind this?

![image](https://github.com/epicmaxco/vuestic-ui/assets/2834599/0633d7ff-1d5e-43fb-842b-4a8f9e301e32)

**Comment by m0ksem** [comment]
Hi, you should make backgroundPrimary `transparent` in colors config.
https://ui.vuestic.dev/styles/colors#color-palette

**Reply by zeknoss** [reply]
Hey @m0ksem , thank you for the answer.
But unfortunately "transparent" isn't accepted as a valid color in the config and compiler complains about it. When I set it as an rgba color with 0 transparency, it still looks the same.

**Reply by m0ksem** [reply]
Try maybe `#ffffff00`?

**Reply by zeknoss** [reply, marked as answer]
That doesn't work either. But when I remove the color-scheme CSS prop from the HTML, the background becomes transparent. Finally I found the culprit.
